components/tooltip/tooltip.js

```diff
@@ -730,9 +730,13 @@ class Tooltip extends RtlMixin(LitElement) {
 			target = ownerRoot.querySelector(targetSelector);
 			target = target || ownerRoot?.host?.querySelector(targetSelector);
 		} else {
-			console.warn('<d2l-tooltip>: missing required attribute "for"');
 			const parentNode = this.parentNode;
 			target = parentNode.nodeType === Node.DOCUMENT_FRAGMENT_NODE ? ownerRoot.host : parentNode;
+
+			// reduce console pollution since Safari + VO prevents inadequate SR experience for tooltips during form validation when using 'for'
```

Comment:
Is it only the `required` case that we care to suppress this for? For example, would we also want to suppress the console warning if its a validation error due to `minlength="2"`, `pattern="[A-Za-z]+"`, etc?

Reply:
Good point. Will update to remove the `required` expression.